packages/account/src/Containers/account.jsx

```diff
@@ -27,9 +28,30 @@ const AccountLogout = ({ logout, history }) => {
     );
 };
 
+const TradingHubLogout = ({ logout }) => {
+    const history = useHistory();
+    return (
+        <div
+            className='dc-vertical-tab__header-account__logout-tab'
+            onClick={() => {
+                logout();
+                history.push(shared_routes.index);
+            }}
+        >
+            <div className='dc-vertical-tab__header-account__logout'>
+                <Icon icon='IcLogout' className='dc-vertical-tab__header-account__logout--icon' />
+                <Text color='general' size='xs' weight='bold'>
```

Comment:
`Text` color should be general by default, you can double check in `Text.jsx` file
```suggestion
                <Text size='xs' weight='bold'>
```